### DISCUSSION TITLE
Working on a full positional visualizer for races and building a local copy of your API

### DISCUSSION THREAD

**Discussion by danilojoncic** [br-g/openf1#12] [Show and tell]
Just wanted to say thank you for the API and the available data for 2023.

**Comment by connor-rose** [comment]
Are you working on a positional visualizer for past races? Like pulling the historical races and displaying via some kind of timeline method to replay old races?

**Reply by danilojoncic** [reply]
exactly, if you have discord share your link for 5 mins and i can show you the progress

**Reply by connor-rose** [reply]
discordapp.com/users/connor.rose

**Comment by adityakotha03** [comment]
Hey looks interesting, does it only show position data or does it also include leaderboard standings and car data during that timeline?
